app/controllers/requests_controller.rb

```diff
@@ -1,8 +1,30 @@
 class RequestsController < ApplicationController
+
+  before_action :set_request, only: :destroy
+
   def create
+    @request.new(request_params)
+    @request.user = current_user
+    @request.event = Event.find(params[:event_id])
+    if @request.save
+      redirect_to user_path(current_user)
+    else
+      render "events/#{params[:event_id]}"
+    end
   end
 
```

Comment:
Do we want an update method?

Reply:
The update method is not necessary, but we need a bunch of others, to confirm the event and modify it, so it probably needs to exist